src/shared/common/components/__tests__/__snapshots__/workstack.spec.jsx.snap

```diff
@@ -156,7 +156,7 @@ Array [
                       <td
                         class="govuk-table__cell"
                       >
-                        is Active
+                        is Active,TEST
```

Comment:
Are these meant to be here?

Reply:
Yes when we specify a renderer of indicator and the value is YES.

Reply:
Thinking about it, it shouldn't, it should only be the active thats shown as its matched the first. Will rectify tomorrow.

Reply:
This has been fixed, thanks for pointing it out 👍🏼